src/applications/vaos/utils/constants.js

```diff
@@ -262,7 +262,7 @@ export const FLOW_TYPES = {
 
 export const TYPE_OF_VISIT = [
   {
-    id: 'office',
+    id: 'clinic',
```

Comment:
Our visit type ids lined up well for phone and telehealth, but we had office instead of clinic. Switching means that we can set kind directly from the `visitType` field in the form.